bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/HBaseRequestAdapter.java

```diff
@@ -43,6 +48,11 @@
  */
 public class HBaseRequestAdapter {
 
+  private final static RequestContext REQUEST_CONTEXT = RequestContext.create(
+      InstanceName.of("", ""),
+      ""
+  );
+
```

Comment:
Why not create a real request context using the BigtableOptions? then you can remove `toMutateRowRequest`